requirements.txt

```diff
@@ -6,4 +6,3 @@ partridge >= 1.1
 pyyaml
 geojson
 jsonschema
-folium
```

Comment:
suggest keeping. there is `import folium` in  "roadwaynetwork.py" and "Roadway Network Search.ipynb"

Reply:
Idea is to separate out all the requirements that are being used vs not...so when GH Actions runs it isn't bogged down and can run efficiently. BUT - it wont pass until I refactor `roadwaynetwork.py` to separate out  viz stuff...which I need to do but was putting off until the other big roadwaynetwork refactor was added. 

But...i can see that it is annoying b/c nothing works w/out it at the moment so I'll add back in.